### DISCUSSION TITLE
ifm3d dump error when uploading ssh keys

### DISCUSSION THREAD

**Discussion by Edumarek123** [ifm/ifm3d#444] [Q&A] [answered]
This is the error that I getting,

Command:
$ echo {} | jq --arg id "$(< ~/.ssh/id_o3r.pub)" '.device.network.authorized_keys=$id' | ifm3d --ip=10.0.1.95 config

Error:
ifm3d error: 100002
Unknown error 100002

pls help me

**Comment by lola-masson** [comment, marked as answer]
Could you share the current configuration of your device (the result of `ifm3d dump`)?

**Reply by Edumarek123** [reply]
{
  "applications": {
    "classes": {
      "mcc": {
        "instanceSchema": {
          "$schema": "http://json-schema.org/draft-07/schema#",
          "additionalProperties": false,
          "properties": {
            "class": {
              "type": "string"
            },
            "configuration": {},
            "data": {},
            "name": {
              "maxLength": 64,
              "type": "string"
            },
            "ports": {},
            "state": {
              "default": "CONF",
              "enum": [
                "CONF",
                "ERROR",
                "RUN"
              ],
              "type": "string"
            }
          },
          "title": "Application parameter schema",
          "type": "object"
        },
        "license": "FREE",
        "minimumFirmwareVersion": {
          "major": 1,
          "minor": 0,
          "patch": 0
        },
        "version": {
          "major": 1,
          "minor": 1,
          "patch": 9
        }
      },
      "ods": {
        "instanceSchema": {
          "$schema": "http://json-schema.org/draft-07/schema#",
          "additionalProperties": false,
          "properties": {
            "class": {
              "type": "string"
            },
            "configuration": {},
            "data": {},
            "name": {
              "maxLength": 64,
              "type": "string"
            },
            "ports": {},
            "state": {
              "default": "CONF",
              "enum": [
                "CONF",
                "ERROR",
                "RUN"
              ],
              "type": "string"
            }
          },
          "title": "Application parameter schema",
          "type": "object"
        },
        "license": "UNLICENSED",
        "minimumFirmwareVersion": {
          "major": 0,
          "minor": 16,
          "patch": 3
        },
        "version": {
          "major": 1,
          "minor": 1,
          "patch": 9
        }
      }
    },
    "instances": {}
  },
  "device": {
    "clock": {
      "currentTime": "1581102646122448928",
      "sntp": {
        "active": true,
        "activeServerAddress": "",
        "activeServerName": "time2.google.com",
        "availableServers": [],
        "systemClockSynchronized": false
      }
    },
    "diagnostic": {
      "confInitStages": [
        "device",
        "ports",
        "applications"
      ],
      "data": {
        "pcicPort": 50009
      },
      "temperatures": [
        {
          "entity": "PLL-therm",
          "value": 44.0
        },
        {
          "entity": "MCPU-therm",
          "value": 44.0
        },
        {
          "entity": "PMIC-Die",
          "value": 100.0
        },
        {
          "entity": "Tboard_tegra",
          "value": 41.0
        },
        {
          "entity": "GPU-therm",
          "value": 42.0
        },
        {
          "entity": "BCPU-therm",
          "value": 44.0
        },
        {
          "entity": "thermal-fan-est",
          "value": 43.0
        },
        {
          "entity": "Tdiode_tegra",
          "value": 41.5
        }
      ],
      "upTime": 12002000000000
    },
    "docker": {
      "insecure-registries": []
    },
    "info": {
      "description": "",
      "device": "0302",
      "features": {},
      "guiSettings": "",
      "name": "vpsolucoes",
      "partNumber": "OVP800",
      "productionState": "AA",
      "serialNumber": "000018429784",
      "vendor": "0001"
    },
    "log": {
      "level": "warn",
      "storage": "volatile"
    },
    "network": {
      "authorized_keys": "",
      "interfaces": {
        "eth0": {
          "ipv4": {
            "address": "10.0.1.95",
            "dns": "10.0.1.255",
            "gateway": "10.0.1.1",
            "mask": 24
          },
          "mac": "48:B0:2D:54:F2:31",
          "networkSpeed": 1000,
          "useDHCP": false
        },
        "eth1": {
          "ipv4": {
            "address": "192.168.42.69",
            "dns": "0.0.0.0",
            "gateway": "0.0.0.0",
            "mask": 24
          },
          "mac": "00:02:01:43:29:04",
          "networkSpeed": 10,
          "useDHCP": false
        }
      }
    },
    "status": "OPERATE",
    "swVersion": {
      "euphrates": "1.32.32",
      "firmware": "1.1.30-1882",
      "kernel": "4.9.140-l4t-r32.4+g8c7b68130d9a",
      "l4t": "r32.4.3",
      "schema": "v1.5.4",
      "tcu": "1.1.1"
    }
  },
  "ports": {
    "port0": {
      "acquisition": {
        "delay": 0,
        "framerate": 10.0,
        "version": {
          "major": 0,
          "minor": 0,
          "patch": 0
        }
      },
      "data": {
        "algoDebugConfig": {
          "channels": [
            {
              "id": "ov9782/calib",
              "type": "once"
            },
            {
              "id": "ov9782/dbg",
              "type": "normal"
            },
            {
              "id": "ov9782/frameInfo",
              "type": "normal"
            },
            {
              "id": "ov9782/ifcustom",
              "type": "normal"
            },
            {
              "id": "ov9782/ifout",
              "type": "normal"
            },
            {
              "id": "ov9782/profile",
              "type": "normal"
            }
          ],
          "version": {
            "major": 1,
            "minor": 1,
            "patch": 9
          }
        },
        "availablePCICOutput": [
          "JPEG_IMAGE",
          "RGB_INFO"
        ],
        "pcicTCPPort": 50010
      },
      "info": {
        "calibration": {
          "version": {
            "major": 0,
            "minor": 1,
            "patch": 1
          }
        },
        "device": "2301",
        "deviceTreeBinaryBlobOverlay": "001-ov9782.dtbo",
        "features": {
          "fov": {
            "horizontal": 127,
            "vertical": 80
          },
          "resolution": {
            "height": 800,
            "width": 1280
          },
          "type": "2D"
        },
        "icc": {
          "version": {
            "major": 0,
            "minor": 0,
            "patch": 0
          }
        },
        "name": "",
        "partNumber": "O3R225",
        "productionState": "AA",
        "sensor": "OV9782",
        "sensorID": "OV9782_127x80_noIllu_Csample",
        "serialNumber": "000018714350",
        "vendor": "0001",
        "version": {
          "major": 11,
          "minor": 1,
          "patch": 0
        }
      },
      "mode": "standard_autoexposure2D",
      "processing": {
        "extrinsicHeadToUser": {
          "rotX": 0.0,
          "rotY": 0.0,
          "rotZ": 0.0,
          "transX": 0.0,
          "transY": 0.0,
          "transZ": 0.0
        },
        "version": {
          "major": 1,
          "minor": 1,
          "patch": 9
        }
      },
      "state": "IDLE"
    }
  }
}

**Reply by Edumarek123** [reply]
I've reinstaled the ifm3d on ubuntu 22.04 and it worked, thanks for the effort
